gui/mutt_window.c

```diff
@@ -769,6 +749,26 @@ void window_redraw(struct MuttWindow *win)
   mutt_refresh();
 }
 
+/**
+ * window_is_focused - Does the given Window have the focus?
+ * @param win Window to check
+ * @retval true Window has focus
+ */
+bool window_is_focused(struct MuttWindow *win)
+{
+  if (!win)
+    return false;
+
+  struct MuttWindow *win_focus = window_get_focus();
+  if (!win_focus)
+    return false;
+
+  if (win_focus == win)
+    return false;
+
+  return true;
+}
```

Comment:
Similarly, here is also logically incorrect. It will return `false` if the window **is** focused. We can simplify the logic down to:

```C
return win_focus == win
```

Reply:
done, thanks